Parse.podspec

```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |s|
   s.name             = 'Parse'
-  s.version          = '1.19.3'
+  s.version          = '1.19.4'
   s.license          =  { :type => 'BSD', :file => 'LICENSE' }
   s.homepage         = 'http://parseplatform.org/'
   s.summary          = 'A library that gives you access to the powerful Parse cloud platform from your iOS/OS X/watchOS/tvOS app.'
   s.authors          = 'Parse Community'
   s.social_media_url = 'https://twitter.com/ParsePlatform'
 
-  s.source           = { :git => 'https://github.com/parse-community/Parse-SDK-iOS-OSX.git', :tag => s.version.to_s }
+  s.source           = { :git => 'https://github.com/adamlevy-emjay/Parse-SDK-iOS-OSX.git', :tag => s.version.to_s }
```

Comment:
This is pointing to your own fork, please change

Reply:
Sorry let me fix it.